### DISCUSSION TITLE
help!! Is the work order I submitted not processed after 1 month?

### DISCUSSION THREAD

**Discussion by lnsane** [community/community#122089] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

 Request Received (#2706571)

**Comment by lnsane** [comment]
```

Thank you for contacting GitHub Support. We wanted to let you know that we've received your message. We are experiencing high volumes and therefore, you may experience longer than normal wait times. In the meantime, you may find answers to commonly asked questions in our community forum or in our documentation.

Ticket ID: 2706571



If you have any additional information or would like to add anything to your initial message, now would be a great time to do so, feel free to reply to this email. If not, then rest assured your request is in the right hands :)

Thank you!
The GitHub Support Team
```

**Comment by lnsane** [comment]
So why not reply?

**Comment by ebndev** [comment]
Hi @lnsane,

This community is managed by GitHub employees, but it is not a support forum. You have done everything necessary on your end by opening a ticket on our [Support page](https://support.github.com/contact).

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. 

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. We appreciate your understanding!
